docs/markdown/authoritative/backend-generic-mysql.md

```diff
@@ -0,0 +1,87 @@
+# Generic MySQL backend
+|&nbsp;|&nbsp;|
+|:--|:--|
+|Native|Yes|
+|Master|Yes|
+|Slave|Yes|
+|Superslave|Yes|
+|Autoserial|Yes (v3.1 and up)|
+|Case|All lower|
+|DNSSEC|Yes (set `gmysql-dnssec`)|
+|Disabled data|Yes (v3.4.0 and up)|
+|Comments|Yes (v3.4.0 and up)|
+|Module name | gmysql|
+|Launch name| gmysql|
+
+**warning**: If using MySQL with 'slave' support enabled in PowerDNS you **must**
+run MySQL with a table engine that supports transactions.
+In practice, great results are achieved with the 'InnoDB' tables. PowerDNS will
+silently function with non-transaction aware MySQLs but at one point this is
+going to harm your database, for example when an incoming zone transfer fails.
+
+The default setup conforms to the following schema:
+
+```
+!!include=../modules/gmysqlbackend/schema.mysql.sql
+```
+
+`zone2sql` with the `--gmysql` flag also assumes this layout is in place.
+
+For full migration notes, please see [Migration](migration.md).
+
+This schema contains all elements needed for master, slave and superslave operation.
+
+When using the InnoDB storage engine, we suggest adding the following lines to
+the 'create table records' command above:
+
```

Comment:
add link to `[Generic SQL backend](backend-generic-sql.md)` as in the goracle docs?